src/presets/cloudflare/runtime/imports/workers.ts

```diff
@@ -0,0 +1,24 @@
+export const env = new Proxy(
+  {},
+  {
+    get(_target, prop) {
+      return globalThis.__env__?.[prop as keyof typeof globalThis.__env__];
```

Comment:
Do we really need a proxy wrapper? (considering `__env__` should be bound to CF proxy itself already and i guess itself is a proxy no?